lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -2396,6 +2453,16 @@ instance LiftHandler ErrNotASequentialWallet where
                 , "Make sure to use a sequential wallet style, like Icarus."
                 ]
 
+instance LiftHandler ErrWithdrawalNotWorth where
+    handler = \case
+        ErrWithdrawalNotWorth ->
+            apiError err403 WithdrawalNotWorth $ mconcat
+                [ "I've noticed that you're requesting a withdrawal from an "
+                , "account that is either empty or doesn't have a balance big "
```

Comment:
it handles also situations, where we got already rewards... saying that account is empty covers this. We don't want to say explicitly something like : `presumably the account was already withdrawn` ?

Reply:
Well, an account could be empty for two reasons: either it hasn't received rewards yet, or it has already withdrawn them. I didn't feel like emphasizing on any of these aspects but we could indeed.